assessment.rb

```diff
@@ -1,8 +1,14 @@
 require 'csv'
 # Assessment answers to:
+Generate a model (`Licensee`) and migration (`db/migrate/<time stamp>_create_licensees.rb`) to hold the data in `data/licensees.csv`.  Run `rake db:migrate` to create the table.
 
+class Licensee < ActiveRecord::Migration
+  def create_table "licensees"
+    surname: CHARACTER VARYING
+    given name: CHARACTER VARYING
+  end
```

Comment:
Having a space in the middle of `given name` confuses Ruby into thinking that it's dealing with two variables.